### ISSUE TITLE
Show Email in Group Membership

### ISSUE THREAD

**Issue by Joe2k** [india-ultimate/hub#175]
Should we show email ids in group memberships which will help people to pay for the right account when there are same multiple names or one person having multiple accounts

**Comment by punchagan** [comment]
:+1: 

I think it makes sense to show the email address in the roster validation page too. Just to be able to see which UC account has been rostered in a team.

**Comment by Joe2k** [comment]
Yep getting lot of queries on this. So will be easy to check themselves

**Comment by Joe2k** [comment]
@punchagan just curious.. Why do we make emails when we sending the list of players to select in group membership?

**Comment by punchagan** [comment]
Because it's easy for anyone to sign up to the Hub and all this data would then be available to them. I didn't want to leak personal info... 

On the rosters page also there's some personal info, but it's only restricted to your own team (unless you're an admin).
